x-pack/platform/plugins/shared/task_manager/server/integration_tests/removed_types.test.ts

```diff
@@ -114,8 +113,20 @@ describe.skip('unrecognized task types', () => {
     taskIdsToRemove.push(notRegisteredTypeId);
 
     // To be sure that the background task that marks removed tasks as unrecognized has run after the tasks were created
-    const runSoonResponse = await taskManagerPlugin.runSoon('mark_removed_tasks_as_unrecognized');
-    expect(runSoonResponse).toEqual({ id: 'mark_removed_tasks_as_unrecognized' });
+    await retry(async () => {
+      try {
+        const runSoonResponse = await taskManagerPlugin.runSoon(
+          'mark_removed_tasks_as_unrecognized'
+        );
+        expect(runSoonResponse).toEqual({ id: 'mark_removed_tasks_as_unrecognized' });
+      } catch (err) {
+        // it's possible the task is already running, so catch those errors and retry
+        // otherwise throw
+        if (!err.message.includes('as it is currently running')) {
```

Comment:
should we bother checking this?  Wondering if this will get flaky if an error message changes in the future ...

Reply:
Sure, I can remove and we can retry in the case of any error.

Reply:
Updated in https://github.com/elastic/kibana/pull/210399/commits/8c6887f873669ba1ac93ac9e91c3f1b3e7ea9ecb